fastplotlib/widgets/image.py

```diff
@@ -483,43 +482,54 @@ def window_funcs(self) -> Dict[str, _WindowFunctions]:
         return self._window_funcs
 
     @window_funcs.setter
-    def window_funcs(self, sa: Dict[str, int]):
-        if sa is None:
+    def window_funcs(self, callable_dict: Dict[str, int]):
```

Comment:
Looks like the type annotation is also leftover from when I only used this for averaging. The type annotation can also be more precise in py3.10+

At the top add the import `from typing import Literal`

and then:

```python
def window_funcs(self, callables_dict: dict[Literal["t", "z"], tuple(callable, int)])
```

I'm curious if this works for dicts now :thinking: 

```python
def window_funcs(self, callables_dict: dict[Literal["t", "z"]: tuple(callable, int)])
```

Reply:
@kushalkolar Can do this, but it might be harder to maintain. Hard coding the accepted keys in two locations (the global variable at the top of the file and in this signature) seems like something we'd want to avoid. 

Also from typing import * is already at the top so I think we are good to go

Reply:
Yea you're right, let's just use str for the key then